ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -561,6 +605,14 @@ export class ObjectDetailComponent implements OnInit {
     }
   }
 
+  checkIfCcColumn(colId: string): boolean {
+    const columnName = this.conv.SpSchema[this.currentObject!.id].ColDefs[colId].Name;
+    
+    return this.conv.SrcSchema[this.currentObject!.id].CheckConstraints.some((cc: ICheckConstraints) => 
```

Comment:
This might not work if this column is new and doesn't exist in source schema

Reply:
addressed the comment